packages/core/src/request-builder/create-request-builder.ts

```diff
@@ -27,11 +27,11 @@ export class CreateRequestBuilder<EntityT extends Entity>
    * Creates an instance of CreateRequestBuilder.
    *
    * @param _entityConstructor - Constructor type of the entity to be created
-   * @param entity - Entity to be created
+   * @param _entity - Entity to be created
    */
   constructor(
     readonly _entityConstructor: Constructable<EntityT>,
-    readonly entity: EntityT
+    readonly _entity: EntityT
```

Comment:
Technically this is a breaking change. Not sure what the best approach here is, but maybe you can add a getter for entity, e. g. `get entity(): EntityT { return this._entity; }` just in case?

Reply:
ok, I just did not want to have the value twice, but with the getter it is save.